dev/parm/config/gfs/config.resources

```diff
@@ -563,11 +564,18 @@ case ${step} in
 
   "ocnanalecen")
     ntasks=16
+    walltime="00:10:00"
     case ${OCNRES} in
       "025")
-        ntasks=40
-        memory="256GB"
-        ;;
+	ntasks=256
+	if [[ "${NMEM_ENS}" -gt 30 ]]; then
+          memory="500GB"
+	  walltime="01:30:00"
+        else
+           ntasks=40
+  ￼        memory="256GB"
+        fi
+	;;
```

Comment:
This is fine.
Please do fix the indentation.

Reply:
fixed!